kolibri/plugins/audio_mp3_render/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "kolibri-audio-mp3-render-plugin",
+  "description": "Audio/MP3 Render Plugin for Kolibri",
+  "private": true,
+  "dependencies": {
+    "html5media": "1.1.8"
+  }
+}
```

Comment:
Opened here: https://github.com/learningequality/kolibri/issues/527 Please add to it!